dash.el

```diff
@@ -2209,6 +2209,11 @@ docstring if none is provided."
          `((-let ,let-bindings ,@body))
        body))))
 
+(defun dash--normalize-arglist (arglist)
+  "Make ARGLIST have the form (MATCHERS...).
+If it is a vector, convert it to a single-matcher arglist."
+  (if (vectorp arglist) (list (append arglist nil)) arglist))
```

Comment:
Can I use `cl-coerce` here? `append` nil seems fragile.

Reply:
`(append x ())` is the canonical way to convert an array into a list, so I wouldn't worry about it. Also, `dash.el` tries to avoid relying on `cl-lib`.